refactor fullscreen handling + unload tab fix

balcy opened the pull request:
fix fullscreen chrome / progressbar visibility for webapps
fix unload function for tabs (fixes https://github.com/ubports/morph-browser/issues/59)

balcy committed:
refactor fullscreen handling (remove additional signal I had previously added)
fix fullscreen chrome / progressbar visibility for webapps
fix unload function for tabs

UniversalSuperBox commented:
Can you add the issues that this fixes to the first post so they're closed automatically? You can do that by editing the post and typing "fixes #1" for example.

UniversalSuperBox reviewed (changes requested):
This does fix the issue, allowing apps which get full screen to move the window controls away! However, just a small change. Everywhere that you comment out code (even if it's less than 10 lines apart), please put the information about why it was commented out.

balcy commented:
ok thanks for the comment. 
Issue https://github.com/ubports/morph-browser/issues/42 is only solved partially, so I've only put the other issue in the first post

balcy committed:
window.currentWebview.fullscreen = false can no longer be used
the property did only exist for oxide (for QtWebEngine it is called 'isFullScreen' and readonly
